.claude/agents/konflux-build-triage.md

```
@@ -0,0 +1,442 @@
+
+### COMPONENT (PIPELINE_TYPE)
+- **Failed task:** `TASK_NAME` (PHASE phase)
+- **Category:** CATEGORY (retryable / not retryable)
+- **Log source:** LOG_SOURCE
+- **Error:** `ERROR_MESSAGE`
+
+<details><summary>Log context</summary>
+
+```
+RELEVANT_LOG_LINES
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)